### PR TITLE
data plumbing draft

### PR DESCRIPTION
**Pull request by SarahU**
# What this does

first notes on data plumbing, taken from a lightning talk

## Our checklist

- [ ] Commit message
- [ ] Tests written
- [ ] For front end changes have you accounted for accessibility?
- [ ] Have you considered performance?

## More information

- [ ] [Github Issue](https://github.com/LadiesOfCodeGroupSessions/loc-digital-garden/issues)

## Screenshots/Videos

_Visuals that may help the reviewer for any applicable front end changes_

### CODE REVIEW

File: src/content/index.md [hunk 1]
```diff
@@ -5,3 +5,5 @@ title: 'Posts'
 [/loc-digital-garden](/loc-digital-garden)
 
 [/terraform-101](/terraform-101)
+
+[/data-plumbing](/data-plumbing)
```
Comment (charlottebrf) [inline src/content/index.md:9]: **discussion:** very exciting to get another post :) 

As it currently stands, this change will add a route but there's no content found. You can see this on the deploy preview site: https://deploy-preview-33--ladies-of-code-digital-garden.netlify.app/data-plumbing


I  would also expect to see the mark down file with the content , e.g. `data-plumbing.md` in [the content folder](https://github.com/LadiesOfCodeGroupSessions/loc-digital-garden/tree/main/src/content). Is this going to be added in this PR?

**Review by charlottebrf — COMMENTED** [review]
Left a question , appreciate this is a draft so perhaps the intention is to add the file later

File: src/content/index.md (hunk 1 above)
Comment (SarahU) [reply src/content/index.md:9]: Oh my gosh Charlotte, I completely missed the actual content! I've added it now. Will double check

Comment (SarahU) [reply src/content/index.md:9]: It seems like it's ok now? Sorry about that! It worked ok locally but I missed the new file in the commit. Love the deployment preview :D

**Review by charlottebrf — APPROVED** [review]

File: src/content/index.md (hunk 1 above)
Comment (charlottebrf) [reply src/content/index.md:9]: LGTM.

File: src/content/data-plumbing.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: "What your plumber can teach you about Data Pipelines"
+description: "What principles of flow, can we apply to tech, from our water management colleagues."
+created_at: 2023-02-19
```
Comment (charlottebrf) [inline src/content/data-plumbing.md:4]: **nitpick:** you may want to update the `created_at` & `updated_at`dates

Comment (SarahU) [reply src/content/data-plumbing.md:4]: Thanks Charlotte. I thought about that but as that's when I actually wrote the content, and didn't change anything, I left them the same

**Comment by SarahU** [comment]
It seems some requirements are not met but it's not totally clear what that is. Something with the Header rules or Pages changed? The details don't specifiy an error

**Review by charlottebrf — APPROVED** [review]

**Comment by charlottebrf** [comment]
> It seems some requirements are not met but it's not totally clear what that is. Something with the Header rules or Pages changed? The details don't specifiy an error

I had an incorrect setting apologies. Should be fixed now & you can merge when you're ready

**Comment by SarahU** [comment]
> > It seems some requirements are not met but it's not totally clear what that is. Something with the Header rules or Pages changed? The details don't specifiy an error
> 
> I had an incorrect setting apologies. Should be fixed now & you can merge when you're ready

Ah, super. Thanks Charlotte!

**Pull request merged**